x-pack/test/spaces_api_integration/common/suites/delete.ts

```diff
@@ -135,17 +141,17 @@ export function deleteTestSuiteFactory(es: any, esArchiver: any, supertest: Supe
     // There were eleven multi-namespace objects.
     // Since Space 2 was deleted, any multi-namespace objects that existed in that space
     // are updated to remove it, and of those, any that don't exist in any space are deleted.
-    const multiNamespaceResponse = await es.search({
+    const { body: multiNamespaceResponse } = await es.search<Record<string, any>>({
       index: '.kibana',
       body: { query: { terms: { type: ['sharedtype'] } } },
     });
-    const docs: [Record<string, any>] = multiNamespaceResponse.hits.hits;
+    const docs = multiNamespaceResponse.hits.hits;
     expect(docs).length(10); // just ten results, since spaces_2_only got deleted
     docs.forEach((doc) => () => {
       const containsSpace2 = doc?._source?.namespaces.includes('space_2');
       expect(containsSpace2).to.eql(false);
     });
-    const space2OnlyObjExists = docs.some((x) => x._id === CASES.SPACE_2_ONLY);
+    const space2OnlyObjExists = docs.some((x) => x._id === CASES.SPACE_2_ONLY.id);
```

Comment:
__note:__ if tests pass with this change then it's probably not used at all....

Reply:
It's hard to prove a negative :). I think we should adjust the query above on line 146 to have add the `size` parameter, so that we at least _attempt_ to retrieve more than 10 objects. ES won't retrieve more than 10 without that parameter set

Reply:
Let me try to make this adjustment then.

Reply:
Done in https://github.com/elastic/kibana/pull/95402/commits/04413a38d056c975a0b84d1a82b29af38f85bf4d